libs/utils/analysis/latency_analysis.py

```diff
@@ -783,6 +784,28 @@ def plotRuntimes(self, task, tag=None, threshold_ms=8, bins=64):
         return stats_df.append(pd.DataFrame(
             stats.values(), columns=['running_time'], index=stats.keys()))
 
+    def plotTaskResidency(self, task):
+        """
+        Plot CPU residency of the specified task
+        This will show an overview of how much time that task spent being
+        active (running, waking up) on each available CPU, in seconds.
```

Comment:
Do we highlight running vs waking up time?
Otherwise I would remove "waking up" because its mainly an "implementation/internal detail"

Reply:
> Do we highlight running vs waking up time?
They're not distinguished in the resulting dataframe. My reasoning is that if a task is waking, it's eating CPU time as well. My plan for this wasn't to create extremely accurate statistics, but rather to get a big picture of the task's lifetime - if you believe something more accurate would actually be useful, I could change that.

Reply:
I would say that we can just avoid to list running, waking up in the documentation.

```diff
- active (running, waking up) on each available CPU, in seconds.
+ active on each available CPU, in seconds.
```